File: app/src/main/java/com/prisyazhnuy/newsapp/NewsApplication.java
package com.prisyazhnuy.newsapp;

import android.app.Application;

//import com.splunk.mint.Mint;
import io.realm.Realm;

/**
 * max.pr on 04.03.2018.
 */

public class NewsApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
//        Mint.initAndStartSession(this, "dc50571a");
    }
}

File: app/src/main/java/com/prisyazhnuy/newsapp/NewsApplication.kt
package com.prisyazhnuy.newsapp

import android.app.Application
import com.prisyazhnuy.newsapp.dataKotlin.database.DataBaseCreator
import io.realm.Realm

/**
 * max.pr on 18.03.2018.
 */
class NewsApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        DataBaseCreator.createDB(this)
    }
}